### PR TITLE
Migrate Mongo query tab to react

### PR DESCRIPTION
**Pull request by vaidankarswapnil**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/854?feature.someFeatureFlagYouMightNeed=true)

https://user-images.githubusercontent.com/81285216/120366692-983cd400-c32d-11eb-8c49-d7b9394d437c.mp4

### CODE REVIEW

File: src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,1132 @@
...
+                    lineNumbers={"on"}
+                    onContentChanged={(newContent: string) => this.onChangeContent(newContent)}
+                    onContentSelected={(selectedContent: string) => this.onSelectedContent(selectedContent)}
+                  />
+                </div>
+              </Fragment>
+              <Fragment>
+                {this.isPreferredApiMongoDB && this.state.sqlQueryEditorContent.length === 0 && (
+                  <div className="mongoQueryHelper">
+                    Start by writing a Mongo query, for example: <strong>{"{'id':'foo'}"}</strong>
```
Comment (victor-meng) [inline src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx:714]: Add space after `<strong>{"{'id':'foo'}"}</strong>`

File: src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,1132 @@
...
+                    or
```
Comment (victor-meng) [inline src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx:715]: Add space after `or`

File: src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,1132 @@
...
+                    <strong>
+                      {"{ "}
+                      {" }"}
+                    </strong>
```
Comment (victor-meng) [inline src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx:719]: Add space after `</strong>` as well.

File: src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx (hunk 1 above)
Comment (vaidankarswapnil) [reply src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx]: Added the space

File: src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx (hunk 2 above)
Comment (vaidankarswapnil) [reply src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx]: Added the space.

File: src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx (hunk 3 above)
Comment (vaidankarswapnil) [reply src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx]: Added the space.

**Pull request merged**